scss/widgets/base/_checkBox.scss

```diff
@@ -22,13 +22,15 @@
   background-position: 0 0;
   background-size: cover;
   background-repeat: no-repeat;
+  flex-shrink: 0;
+  max-height: 100%;
 }
 
 .dx-checkbox-container {
   height: 100%;
   width: 100%;
-  display: inline-block;
-  vertical-align: middle;
+  display: flex;
+  column-gap: 5px;
```

Comment:
`column-gap` is not supported on ios safari 14.4-,  android 4.4.4-, opera mobile and so on. See [this link](https://caniuse.com/?search=column-gap). Can we style it another way? For example, we `padding: 2.5px` here and `margin: 2.5px` for childs?

Reply:
container only have icon and optional text. looks like we can just add `padding-left` for text element